app.js

```diff
@@ -1,29 +1,16 @@
 const express = require('express');
 const app = express();
 const {getCategories} = require('./controllers/categories-controllers');
+const {getReviews, getReviewById} = require('./controllers/reviews-controllers')
 
 app.use(express.json());
 
 app.get('/api/categories', getCategories);
+app.get('/api/reviews', getReviews);
+app.get('/api/reviews/:review_id', getReviewById)
 
 // error handling
 
-// psql errors
-app.use((err, req, res, next) => {
-    if (err.code === '22P02') {
-        res.status(400).send('baaad request x');
-    } else {
-        next(err);
-    }
-})
-// custom errors
-app.use((err, req, res, next) => {
-    if (err.status && err.msg) {
-        res.status(err.status).send({msg: err.msg});
-    } else {
-        next(err);
-    }
-})
```

Comment:
### done for deleting these until they are necessary later - please keep alterations to tickets on the relevant branch - this can easily get confusing later on when changes for other tickets are included